internal/circle/service_test.go

```diff
@@ -258,17 +258,4 @@ func Test_Service_allMethods(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, &Transfer{ID: "transfer-id"}, res)
 	})
-
-	t.Run("GetWalletByID", func(t *testing.T) {
-		mCircleClient := NewMockClient(t)
-		mCircleClient.
-			On("GetWalletByID", ctx, "wallet-id").
-			Return(&Wallet{WalletID: "wallet-id"}, nil).
-			Once()
-		svc := createService(t, mCircleClient)
-
-		res, err := svc.GetWalletByID(ctx, "wallet-id")
-		assert.NoError(t, err)
-		assert.Equal(t, &Wallet{WalletID: "wallet-id"}, res)
-	})
```

Comment:
I think it's worth adding the two ned methods here, just to make sure they're being properly called.

Reply:
Addressed in 7e14a61e48b93b4d67935be63c2622ab8d4f8594